### PR TITLE
feat(mobile): monitor ANR errors for Android apps

### PR DESCRIPTION
**Pull request by ghost**
This PR covers doc updates for monitoring ANR errors (Android only).

Jira: NR-279051

### CODE REVIEW

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+* Device and app scope:
+    * New Relic reports from devices with the New Relic agent installed, regardless of app source.
+    * Google Play Console reports exclusively from certified devices with apps downloaded through the Play Store.
+
+For more information, see the [Google Play documentation](https://developer.android.com/topic/performance/vitals).
+
+## View ANRs in New Relic [#view-anr-in-new-relic]
+
+By default, our Android agent collects and reports ANR errors to New Relic. You can find ANR data in two places:
+    * On the Summary page, there's a widget called <DoNotTranslate>**ANR rate**</DoNotTranslate> displaying the percentage of errors.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx:34]: Give more info on what the actual widget displays

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Our Android agent reports ANR errors as []`MobileApplicationExit`](linktodatadictionary) events, which leverage Android’s [ApplicationExitInfo API](https://developer.android.com/reference/android/app/ApplicationExitInfo) to monitor and report on the different causes for app termination. The agent reads the `ApplicationExitInfo` API on the next app launch and asynchronously sends `MobileApplicationExit` events to New Relic for each app exit in the history.
+
+### Understanding ANR discrepancies between New Relic and Google Play Console [#new-relic-vs-google-play]
+
+You might encounter variations in ANR counts and rates between New Relic and Google Play Console. This is due to fundamental differences in the data collection methods:
+
+* Android version coverage:
+    * New Relic focuses on ANRs from devices running Android 11 or higher.
+    * Google Play Console captures ANRs across all Android versions.
+* Device and app scope:
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx:25]: @silpau-pm Hi!

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+tags:
+  - Mobile monitoring
+  - Mobile monitoring UI
+  - Application not responding
+  - Errors
+metaDescription: Use New Relic's Application Not Monitoring UI page to track and analyze ANR errors. 
+freshnessValidatedDate: 2024-06-24
+---
+
+In Android mobile apps, [Application Not Responding (ANR) errors](https://developer.android.com/topic/performance/vitals/anr) are triggered when the main UI thread of an application is blocked for more than five seconds.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx:12]: @silpau-pm hello

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+You might encounter variations in ANR counts and rates between New Relic and Google Play Console. This is due to fundamental differences in the data collection methods:
+
+* Android version coverage:
+    * New Relic focuses on ANRs from devices running Android 11 or higher.
+    * Google Play Console captures ANRs across all Android versions.
+* Device and app scope:
+    * New Relic reports from devices with the New Relic agent installed, regardless of app source.
+    * Google Play Console reports exclusively from certified devices with apps downloaded through the Play Store.
+
+For more information, see the [Google Play documentation](https://developer.android.com/topic/performance/vitals).
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx:29]:
```suggestion
    * New Relic report from devices with the New Relic agent installed, regardless of app source.
    * Google Play Console reports exclusively from certified devices with apps downloaded through the Play Store.

For more information, see the [Google Play documentation](https://developer.android.com/topic/performance/vitals).
```

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  - Errors
+metaDescription: Use New Relic's Application Not Monitoring UI page to track and analyze ANR errors. 
+freshnessValidatedDate: 2024-06-24
+---
+
+In Android mobile apps, [Application Not Responding (ANR) errors](https://developer.android.com/topic/performance/vitals/anr) are triggered when the main UI thread of an application is blocked for more than five seconds.
+
+## How ANR errors are reported in New Relic [#anr-reporting]
+
+Our Android agent reports ANR errors as []`MobileApplicationExit`](linktodatadictionary) events, which leverage Android’s [ApplicationExitInfo API](https://developer.android.com/reference/android/app/ApplicationExitInfo) to monitor and report on the different causes for app termination. The agent reads the `ApplicationExitInfo` API on the next app launch and asynchronously sends `MobileApplicationExit` events to New Relic for each app exit in the history.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx:16]:
```suggestion
Our Android agent reports ANR errors as [`MobileApplicationExit`](linktodatadictionary) events, which leverage Android’s [ApplicationExitInfo API](https://developer.android.com/reference/android/app/ApplicationExitInfo) to monitor and report on the different causes for app termination. The agent reads the `ApplicationExitInfo` API on the next app launch and asynchronously sends `MobileApplicationExit` events to New Relic for each app exit in the history.
```

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+<img
+    title="anr rate chart"
+    alt="screenshot of anr rate chart"
+    src={mobileAnrRateChart}
+/>
+
+The <DoNotTranslate>**ANR rate by day**</DoNotTranslate> chart shows the percentage of unique users that experienced an ANR over time as a percentage of all unique users.
+
+Note: Irrespective of the time period selected, this chart would show the average ANR rate for the last 28 days.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx:90]:
```suggestion
Note: Regardless of the time selected, this chart will show the average ANR rate for the last 28 days.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/maintenance/configure-settings-for-mobile-monitoring.mdx [hunk 7]
```diff
@@ -122,6 +124,25 @@ To view your application token:
 1. Go to <DoNotTranslate>**[one.newrelic.com > All Capabilities](https://one.newrelic.com/all-capabilities)**</DoNotTranslate>, click <DoNotTranslate>**Mobile**</DoNotTranslate>, and select a monitored app.
 2. Select <DoNotTranslate>**Application**</DoNotTranslate> or <DoNotTranslate>**Installation**</DoNotTranslate>.
 
+## Server-side configuration [#configure-server-side-settings]
+
+Server-side mobile agent configuration provides a mechanism to remotely manage and adjust the behavior of mobile agents deployed on user devices. This functionality empowers you to dynamically control monitoring settings without requiring a new app store deployment.
+
+The following setting can be currently modified remotely on the **Application** page:
+ * Application Not Responding (ANR) error tracking (Android only): Enables tracking and reporting of the reasons behind ANR errors. This functionality is enabled by default and requires Android agent 7.4.0 or higher.
```
Comment (paperclypse) [inline src/content/docs/mobile-monitoring/new-relic-mobile/maintenance/configure-settings-for-mobile-monitoring.mdx:132]:
```suggestion
 * Application Not Responding (ANR) error tracking (Android only): Enables tracking and reporting of the reasons behind ANR errors. This functionality is enabled by default and requires Android agent 7.4.1 or higher.
```

**Pull request merged**